.github/workflows/pinot_compatibility_tests.yml

```diff
@@ -48,7 +48,14 @@ jobs:
           NEW_COMMIT: ${{ github.event.inputs.newCommit }}
           WORKING_DIR: /tmp/compatibility-verifier
           TEST_SUITE: ${{ matrix.test_suite }}
-          MAVEN_OPTS: -Xmx2G -DskipShade -DfailIfNoTests=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 -Dmaven.wagon.http.retryHandler.count=3 -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false
+          MAVEN_OPTS: -Xmx2G -DskipShade -DfailIfNoTests=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=25 \
```

Comment:
is this multi-line with backslash the correct format? 
```
MAVEN_OPTS: >
    -Xmx2G -DskipShade -DfailIfNoTests=false -Dmaven.wagon.httpconnectionManager.ttlSeconds=25
    -Dmaven.wagon.http.retryHandler.count=3 -Dhttp.keepAlive=false -Dmaven.wagon.http.pool=false
    ... 
```

Reply:
You may also need to modify the script a bit to let it trigger the job once to kick off the action tasks

Reply:
good catch. how do I force it to run?

Reply:
Just add a newline in pom.xml should trigger it. You can revert it back once the test is done

Reply:
running now

Reply:
running now